### DISCUSSION TITLE
ip/bilibili可以，但是ip/bilibili/new方法不允许

### DISCUSSION THREAD

**Discussion by IshuaWang** [imsyy/DailyHotApi#84] [Q&A]
ip/bilibili:
{
    "code": 200,
    "name": "bilibili",
    "title": "哔哩哔哩",
    "type": "热门榜",
    "description": "你所热爱的，就是你的生活",
    "params": {
        "type": {
            "name": "排行榜分区",
            "type": {
                "0": "全站",
                "1": "动画",
                "3": "音乐",
                "4": "游戏",
                "5": "娱乐",
                "36": "科技",
                "119": "鬼畜",
                "129": "舞蹈",
                "155": "时尚",
                "160": "生活",
                "168": "国创相关",
                "181": "影视",
                "188": "数码"
            }
        }
    },
    "link": "https://www.bilibili.com/v/popular/rank/all",
ip/bilibili/new:
{
    "code": 405,
    "message": "Method Not Allowed"
}
这是怎么回事呢？

**Comment by imsyy** [comment]
https://blog.imsyy.top/posts/2024/0408#%E8%8E%B7%E5%8F%96%E6%9C%80%E6%96%B0%E6%95%B0%E6%8D%AE

**Reply by IshuaWang** [reply]
我一直没看懂，web的端docker里面环境变量是API还是VITE_GLOBAL_API。我试了两个好像web端都没有结果，不知道应该怎么正确设置才能在web端调用api。
